client/jetpack-connect/authorize-form.jsx

```diff
@@ -90,8 +90,11 @@ class JetpackConnectAuthorizeForm extends Component {
 		);
 	}
 
-	isWCS() {
-		return 'woocommerce-services' === this.props.jetpackConnectAuthorize.queryObject.from;
+	isWoo() {
+		const wooSlugs = [ 'woocommerce-setup-wizard', 'woocommerce-services' ];
+		const jetpackConnectSource = this.props.jetpackConnectAuthorize.queryObject.from;
```

Comment:
I see that `this.props.jetpackConnectAuthorize.queryObject.from;` is from the original line, but might it be beneficial to refactor this to use [`_.get`](https://lodash.com/docs/4.17.4#get)? `get` would handle any missing part of the chain by returning `undefined` instead of throwing a reference error.